glossary.yml

```diff
@@ -1879,6 +1933,13 @@
       A user interface that relies on windows, menus, pointers, and other
       graphical elements, as opposed to a [command-line interface](#cli)
       or voice-driven interface.
+  pt:
+    term: "interface gráfica de usuário"
+    acronym: "GUI"
```

Comment:
@zkamvar the translation to Portuguese is all revised, I'm just not sure if the acronyms repeats or appears once per terms.

Reply:
That's a good question. I think we need to keep the acronyms in their separate languages because they can be different (or non-existent in the case of languages without alphabets). 

Basically, I would allow it.

Reply:
Thanks!